### PR TITLE
CB2-9317: Generate welsh TRL PRS certificate

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## Generate welsh certificate trailer PRS

This change is the changes to the cert gen config to allow the PRS TRL data to flow through into the document generation service.
[link to ticket number](https://dvsa.atlassian.net/browse/CB2-9317)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

**Review by kcarr-bjss — DISMISSED** [review]

File: src/services/CertificateGenerationService.ts [hunk 1]
```diff
@@ -104,6 +104,7 @@ class CertificateGenerationService {
       trl_fail: config.documentNames.vtg30,
       trl_fail_bilingual: config.documentNames.vtg30_bilingual,
       trl_prs: config.documentNames.trl_prs,
+      trl_prs_bilingual: config.documentNames.trl_prs_bilingual,
```
Comment (cb-cs) [inline src/services/CertificateGenerationService.ts:107]: Can you add some unit tests for a TRL PRS BILINGUAL please.

File: tests/unit/certGen.unitTest.ts [hunk 2]
```diff
@@ -3900,6 +3900,167 @@ describe("cert-gen", () => {
                     "utf8"
                 )
             );
+            context("and the TRL has defects that is rectified at the test", () => {
+                const trlPRS = JSON.parse(event.Records[3].body);
+
+                context("and the test station is not in wales", () => {
+                    it("should return a TRL PRS certificate with the defects array populated", async () => {
+                            const expectedResult = {
+                                "DATA": {
```
Comment (cb-cs) [inline tests/unit/certGen.unitTest.ts:3909]: Can you remove the unnecessary quoted keys in the new tests as per TSLint.

File: tests/resources/queue-event-prs.json [hunk 3]
```diff
@@ -29,6 +29,16 @@
       "eventSource": "aws:sqs",
       "eventSourceARN": "arn:aws:sqs:eu-west-2:006106226016:cert-gen-q",
       "awsRegion": "eu-west-2"
+    },
+    {
+      "messageId": "98a8d895-429d-48d4-8ac6-3a430aab75cp",
+      "receiptHandle": "AQEBJcBvTRZ1W2LSaUJ0g0ELXlqA8WCL4zJxO63wu0YOVhx44xxxPhsnc+/Q9+1vOPYO+3HupEjXzGRSvfPY5rEEJkgCJe4/RQ+q2kU5LsmJEr1qE/CTdIYe5X/75XeMQ523KKpdNsD9tRhyvEpPpSu50byGbz7J0JyR6lu1E6Q4YuB4QNm+ev1obPMLdEt8RhgvIi/NfEfQf0L1r3TPi3wLho1R61PllPm27He8/1CjCnMyWBzgX+DCjJ7vyRXObMZ/MbhMBKbYpeTcejsKpYX//PPr1yvldp1YPC0wPKp+iqmWxoDDeHXbo8xYRFXDA8rnY5RfkwxxffH7o534vYn8FCZEtqybQuo7pumu6Ah9PsC05tP38syU71ltasljGIA35BgCdSO+9r5rTaBnbO9++Q==",
```
Comment (cb-cs) [inline tests/resources/queue-event-prs.json:35]: Can you update the prs flag on the defect to true please, as there is only 1 defect and the test result is prs, if it was false the result would be fail.

**Review by cb-cs — DISMISSED** [review]

**Review by kcarr-bjss — DISMISSED** [review]

File: src/services/CertificateGenerationService.ts [hunk 4]
```diff
@@ -95,6 +95,7 @@ class CertificateGenerationService {
       trl_fail: config.documentNames.vtg30,
       trl_fail_bilingual: config.documentNames.vtg30_bilingual,
       trl_prs: config.documentNames.trl_prs,
+        trl_prs_bilingual: config.documentNames.trl_prs_bilingual,
```
Comment (cb-cs) [inline src/services/CertificateGenerationService.ts:98]: For some reason it's moved this over 2 spaces, can you fix the formatting please (it did the same for me)

File: src/models/Enums.ts [hunk 5]
```diff
@@ -149,6 +149,8 @@ export const AVAILABLE_WELSH = {
         "hgv_fail",
         "trl_fail",
         "psv_fail",
-        "hgv_prs"
+        "hgv_prs",
+        "psv_fail",
```
Comment (cb-cs) [inline src/models/Enums.ts:153]: They rebase has duplicated the psv_fail, can you remove please.

**Review by kcarr-bjss — APPROVED** [review]

**Review by cb-cs — APPROVED** [review]

**Pull request merged**